types/jasmine/jasmine-tests.ts

```diff
@@ -1355,6 +1355,80 @@ describe("jasmine.arrayWithExactContents", () => {
     });
 });
 
+describe("jasmine.mapContaining", () => {
```

Comment:
These tests fail at runtime. Any chance you can give me things that work at runtime so I can verify the types are right?

Reply:
![image](https://user-images.githubusercontent.com/7110657/116138516-e877a500-a6a2-11eb-82d5-5a33e6bdb7a2.png)